server/services/dailymotion.js

```diff
@@ -0,0 +1,54 @@
+const URL = require("url");
+const axios = require("axios");
+const ServiceAdapter = require("../serviceadapter");
+const { InvalidVideoIdException } = require("../exceptions");
+const Video = require("../../common/video");
+
+class DailyMotionAdapter extends ServiceAdapter {
+  static SERVICE_ID = "dailymotion";
```

Comment:
I'm getting a syntax error here for some reason

Reply:
```
/home/carson/Documents/code/opentogethertube/server/services/dailymotion.js:8
  static SERVICE_ID = "dailymotion";
                    ^

SyntaxError: Unexpected token =
    at new Script (vm.js:80:7)
    at createScript (vm.js:274:10)
    at Object.runInThisContext (vm.js:326:10)
    at Module._compile (internal/modules/cjs/loader.js:664:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/carson/Documents/code/opentogethertube/server/infoextractor.js:3:28)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/carson/Documents/code/opentogethertube/roommanager.js:6:21)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
```

Reply:
What version of node are you running?

Reply:
`v10.15.3`, but I also tried with `v12.18.3` and I got the same thing. When did this get added to node?

Reply:
> When did this get added to node?

I can't find anything referencing that it did get added to node. In fact, all I can find is documentation saying that static class properties are not yet implemented. So I tried to run the code in 2 more environments, both are separate Linux boxes (I developed on a Windows PC so far) and I always get the same result. I used node v12.18 in all cases.

I also made a mini test script, which also works wherever I try to run it. I also tried this one in a `node:lts` docker container. Same result.

```js
class TestClass {
  static foo = "bar";
}

console.log(TestClass.foo);
```
```
$ node --version
v12.18.2
$ node test.js
bar
```

The thing is that, even though I don't seem to have a problem using them in node lts, static class properties appear to be not yet officially implemented. The `ServiceAdapter` classes don't really depend on `SERVICE_ID` to be a static member. I made it one for convenience in case we need it further down the line but I didn't use it as such yet. So I guess I'll just make it an instance member to be safe.